MyBus/MyBus/Model/Road/MapBusRoad.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  MapBusRoad.swift
+//  MyBus
+//
+//  Created by Lisandro Falconi on 5/9/16.
+//  Copyright © 2016 Spark Digital. All rights reserved.
+//
+
+import Foundation
+import Mapbox
+
+class MapBusRoad: NSObject {
+    var markerList : [MGLPointAnnotation] = [MGLPointAnnotation]()
+    var polyLineList : [MGLPolyline] = [MGLPolyline]()
+
+    func addBusRoadOnMap(roadResult : RoadResult) -> MapBusRoad
+    {
+        if(roadResult.roadResultType == 0)
```

Comment:
You can replace this condition defining a new method inside RoadResult class making it easier to read and assigning the responsibility to the object (the idea here is to follow the "tell don't ask principle" - More details: http://martinfowler.com/bliki/TellDontAsk.html ):

```Swift
....
class RoadResult: NSObject {
.....
    func isSingleRoute() -> Bool
    {
        return (self.roadResultType == 0)
    }
.....
```

Then this line should be replaced by:

```Swift
if (roadResult.isSingleRoute())
```